src/MoBi.Assets/AppConstants.cs

```diff
@@ -1392,8 +1392,10 @@ public static class Captions
          public static readonly string AddValuePoint = "Add Point";
          public static readonly string UseDerivedValues = "Use derivative values";
          public static readonly string OffsetObjectPath = "Path to offset object";
+         public static readonly string XArgumentObjectPath = "Path to x-argument object";
          public static readonly string TableObjectPath = "Path to table object";
          public static readonly string TableFormulaWithOffset = "Table Formula with Offset";
+         public static readonly string TableFormulaWithXArgument = "Table Formula with X-Argument";
          public static readonly string VariableName = "Variable name";
```

Comment:
please use either **_x_** or **_X_** in both `XArgumentObjectPath `and `TableFormulaWithXArgument`

Reply:
Not sure I understand?

Reply:
```
public static readonly string XArgumentObjectPath = "Path to x-argument object";
public static readonly string TableFormulaWithXArgument = "Table Formula with X-Argument";

```
Just write `x-argument` consistently (either **x-arg** or **X-Arg** in both cases)

Reply:
Oh I see. Will do and also correct those other typos